apps/studio/src/types/growthbook.ts

```diff
@@ -0,0 +1,3 @@
+export interface GrowthbookAttributes {
+  email: string
```

Comment:
[question] why use `email` as idnetifier over something likle `uuid`

Reply:
Hmm we don't know the user's `uuid` until they authenticated with Singpass, and we maintain that we are identifying users on Studio purely using their emails